cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -112,6 +122,12 @@ def get_data(self, options, feature_id) -> Pageable[RegulatoryEffectObservation]
             reg_effects = RegEffectSearch.source_search(feature_id)
         else:
             reg_effects = RegEffectSearch.source_search_with_private(feature_id, self.request.user.all_experiments())
+
+
+        if options.get("sig_only"):
```

Comment:
`options.get("sig_only")` should be passed in to `RegEffectSearch.source_search_public`, `RegEffectSearch.source_search`, and `RegEffectSearch.source_search_with_private` so this logic can happen in the ViewModel instead of the view.